src/Microsoft.DotNet.Build.Tasks/Targets/PerfTesting.targets

```diff
@@ -11,17 +11,32 @@
   <!-- Perf Runner NuGet package paths -->
   <PropertyGroup>
     <XunitPerfRunnerPackageId>Microsoft.DotNet.xunit.performance.runner.Windows</XunitPerfRunnerPackageId>
-    <XunitPerfRunnerPackageVersion>1.0.0-alpha-build0012</XunitPerfRunnerPackageVersion>
+    <XunitPerfRunnerPackageVersion>1.0.0-alpha-build0014</XunitPerfRunnerPackageVersion>
     <XunitPerfRunnerPackageDir>$(PackagesDir)$(XunitPerfRunnerPackageId)/$(XunitPerfRunnerPackageVersion)</XunitPerfRunnerPackageDir>
-    <XunitPerfRunnerPath>$(XunitPerfRunnerPackageDir)/tools/xunit.performance.run.exe</XunitPerfRunnerPath>
+    <XunitPerfRunnerPath>$(TestPath)$(DebugTestFrameworkFolder)/xunit.performance.run.exe</XunitPerfRunnerPath>
   </PropertyGroup>
 
   <!-- Perf Analysis NuGet package paths -->
   <PropertyGroup>
     <XunitPerfAnalysisPackageId>Microsoft.DotNet.xunit.performance.analysis</XunitPerfAnalysisPackageId>
-    <XunitPerfAnalysisPackageVersion>1.0.0-alpha-build0012</XunitPerfAnalysisPackageVersion>
+    <XunitPerfAnalysisPackageVersion>1.0.0-alpha-build0014</XunitPerfAnalysisPackageVersion>
     <XunitPerfAnalysisPath>$(PackagesDir)$(XunitPerfAnalysisPackageId)/$(XunitPerfAnalysisPackageVersion)/tools/xunit.performance.analysis.exe</XunitPerfAnalysisPath>
   </PropertyGroup>
+  
+  <!-- Copy over the performance runners to the test directory-->
+  <Target Name="CopyXunitExecutionDesktop" BeforeTargets="RunTestsForProject" Condition="'$(RunPerfTestsForProject)' == 'true'">
+    <ItemGroup>
+	  <PerfRunners Include="$(XunitPerfRunnerPackageDir)/tools/xunit.performance.run.exe" />
+	  <PerfRunners Include="$(XunitPerfRunnerPackageDir)/tools/xunit.performance.metrics.dll" />
+	  <PerfRunners Include="$(XunitPerfRunnerPackageDir)/tools/xunit.performance.logger.exe" />
+	  <PerfRunners Include="$(XunitPerfRunnerPackageDir)/tools/ProcDomain.dll" />
+	  <PerfRunners Include="$(XunitPerfRunnerPackageDir)/tools/Microsoft.Diagnostics.Tracing.TraceEvent.dll" />
+	  <PerfRunners Include="$(XunitPerfRunnerPackageDir)/tools/*/*.*" />
+	</ItemGroup>
+    <Copy SourceFiles="@(PerfRunners)"
+          DestinationFiles="@(PerfRunners->'$(TestPath)$(DebugTestFrameworkFolder)\%(RecursiveDir)%(Filename)%(Extension)')"
```

Comment:
It reacted oddly with the wildcard folder inclusions for PerfRunners e.g. not copying subdirectories properly. I didn't want to copy xunit.core from the xunit.performance,runners directory so I couldnt just include all of it with "tools/**/*.*"s, I had to pick and choose.

I've never made an msbuild target before though, so I expect this could be optimized.